consolidator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:11
+FROM mosipdev/openjdk-17-jre:latest
```

Comment:
why the base image is from mosipdev ?

Reply:
Devops team suggested this image. They are recommending mosipdev images to have light-weighted image avoiding unnecessary packages from the actual image.